### PR TITLE
Implemented JUnit test for DoneCommand

### LINKED ISSUE #85: Add JUnit testing for Done Command Function

### PR DESCRIPTION
**Pull request by yAOwzers**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,4 +43,5 @@ checkstyle {
 
 run{
     standardInput = System.in
+    enableAssertions = true
```
Comment (franceneee) [inline build.gradle:46]: oh yay

File: docs/UserGuide.md [hunk 2]
```diff
@@ -29,7 +29,11 @@
 
 ## Introduction
 
+<<<<<<< HEAD
```
Comment (franceneee) [inline docs/UserGuide.md]: I think you should remove this line...

File: docs/UserGuide.md [hunk 3]
```diff
@@ -29,7 +29,11 @@
...
+Zer0Note is a notetaking and organisation application that combines the ease of use and feature set of graphical tools like OneNote, Evernote, Notion etc. with the interaction speed of command-line based tools like vim and emacs. The target audience is students with intermediate to advanced proficiency with computers. 
+=======
 Zer0Note is a note taking and organisation application that combines the ease of use and feature set of graphical tools with the interaction speed of command-line based tools like vim and emacs.
+>>>>>>> bf8e64bcf963cf37054360f03fe9be186d709457
```
Comment (franceneee) [inline docs/UserGuide.md]: I think this should be removed ?

File: src/test/java/seedu/duke/userinterface/command/MarkAsDoneTest.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    void execute_MarkAsDoneTwo_IndexOutOfBound() {
+        AppState appState = new AppState();
+
+        String input = "add /tTask1 /by12-10-2020 2252";
+        AddCommandTimetableMode addTimetableMode = new AddCommandTimetableMode(input, appState);
+        addTimetableMode.execute();
+
+        assertThrows(IndexOutOfBoundsException.class, () -> { appState.getTaskList().markAsDone(2); });
+    }
+}
```
Comment (franceneee) [inline src/test/java/seedu/duke/userinterface/command/MarkAsDoneTest.java]: Good job!

**Review by franceneee — CHANGES_REQUESTED** [review]
I think there are lines you missed out during merge conflict.. haha... Please fix the checkstyle issues as well!

File: docs/UserGuide.md (hunk 2 above)
Comment (yAOwzers) [reply docs/UserGuide.md]: OMG LOL SORRY

**Review by franceneee — APPROVED** [review]

**Review by Lusi711 — APPROVED** [review]

**Pull request merged**